defaults/main.yml

```diff
@@ -140,11 +140,12 @@ tinc_ifupdown_interface:
   auto_ifup: False
   force: True
   options: |
+    {% if ansible_local.init != 'systemd' %}
     tinc-net {{ tinc_network }}
     tinc-chroot yes
     tinc-mlock yes
     tinc-user {{ tinc_user }}
-    {% if tinc_connection_type == 'static' %}
+    {% elif tinc_connection_type == 'static' %}
```

Comment:
Looks like it's almost over. :-) With this change you won't correctly set up a static network interface for the `tinc` bridge. You should revert `elif` to `if` here and add `{% endif %}` above it, so that above DHCP-related options only are enabled on hosts that don't have `systemd`.

After this change it should be all good. I'll need to look into Travis test some more, but that's fol later.